java/test/org/openqa/selenium/grid/node/config/NodeOptionsTest.java

```diff
@@ -148,7 +149,7 @@ boolean isDownloadEnabled(WebDriverInfo driver, String customMsg) {
               reported.add(caps);
               return Collections.singleton(HelperFactory.create(config, caps));
             });
-    String expected = driver.getDisplayName();
+    String expected = "Edge".equals(driver.getDisplayName()) ? Browser.EDGE.browserName() : driver.getDisplayName();
```

Comment:
This test was broken when I got here. For **Microsoft Edge**, the display name is `Edge`, but the browser name is `MicrosoftEdge`. Consequently, the expected value would never be match.